docs/src/conf.py

```diff
@@ -42,9 +42,10 @@
 # ones.
 extensions = [
     "sphinx.ext.autodoc",
+    "sphinx.ext.intersphinx",
     "sphinx.ext.napoleon",
+    "sphinx.ext.viewcode",
```

Comment:
maybe you add into your final commit message that you add viewcode? or is this related to the linting

Reply:
No sorry, I have to add this.

Reply:
This is not related to the linting, right? So I wondered if you can just mention this in your commit message when you squash, that you add viewcode?

Reply:
Ah okay sorry it is already two commits. Okay forget what I said. Everything fine.